### DISCUSSION TITLE
column.textWrap 被移除了吗? 没有生效

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#42641] [Q&A]
https://codepen.io/valar123/pen/ZEqPVmZ?editors=001

```jsx
  {
    width: 200,
    textWrap: 'word-break',
  },
```

长文本还是没换行

**Comment by kiner-tang** [comment]
请问你是在哪里看到可以使用这个属性的呢？我看了一下，从3.x ~ 5x,好像都没有暴露这个属性出来。

**Reply by vaynevayne** [reply]
https://github.com/ant-design/ant-design/issues/5753

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/17284 这是最后的实现方式。

**Reply by vaynevayne** [reply]
意思是换成了 ellipsis 吗?
